ONSdigital/dp-frontend-renderer
# Added missing 'Back to table of contents' link to Previous Versions

### What

Added missing 'Back to table of contents' link to Previous Versions section on the CMD dataset page.

### How to review

- Navigate to the most recent version of a dataset page; ensure that there are at least two versions of that dataset available. 
- Navigate to the 'Previous Versions' section and see if the link, 'Back to table of contents', is rendered. On click, it should scroll back to the ToC at the top of the page.

### Who can review

Anyone
